index.html

```diff
@@ -1,165 +1,232 @@
+          <li class="tags-elements">bootstrap</li>
+          <li class="tags-elements">Ruby</li>
         </ul>
-
+        <a class="button" href="">See Project</a>
+      </section>
+      <section class="card-works">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="tags">
+          <li class="tags-elements">html</li>
+          <li class="tags-elements">bootstrap</li>
+          <li class="tags-elements">Ruby</li>
+        </ul>
+        <a class="button" href="">See Project</a>
+      </section>
+      <section class="card-works">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="tags">
+          <li class="tags-elements">html</li>
+          <li class="tags-elements">bootstrap</li>
+          <li class="tags-elements">Ruby</li>
+        </ul>
+        <a class="button" href="">See Project</a>
+      </section>
+      <section class="card-works">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="tags">
+          <li class="tags-elements">html</li>
+          <li class="tags-elements">bootstrap</li>
+          <li class="tags-elements">Ruby</li>
+        </ul>
+        <a class="button" href="">See Project</a>
+      </section>
+    </section>
+    <section class="about">
+      <h1>About me</h1>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a class="button" href="">Get My Resume</a>
+      <span class="horizontal-line"></span>
+      <ul class="about-list">
+        <li id="diamond">
+          Languages
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </li>
+        <li id="square">
+          Frameworks
+          <ul>
+            <li>bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CopyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li id="circle">
+          Skills
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section class="contact-form-section">
+      <h1 class="contact-description">
```

Comment:
- [ ] Kindly change the `h1` tag into` h2` .... `h6`, I always the best practice to only have one h1 in your index.html

Reply:
Hi, according to this from a Microverse lesson http://diveintohtml5.it/semantics.html#:~:text=And%20in%20HTML5%2C%20each%20section%20can%20have%20its%20own%20%3Ch1%3E%20element.
in HTML5 each section can have its own h1 element